lux/_config/config.py

```diff
@@ -358,9 +359,10 @@ def set_executor_type(self, exe):
             from lux.executor.SQLExecutor import SQLExecutor
 
             self.executor = SQLExecutor()
-        else:
+        elif exe == "Pandas":
             from lux.executor.PandasExecutor import PandasExecutor
 
+            self.SQLconnection = ""
             self.executor = PandasExecutor()
 
```

Comment:
maybe worthwhile to throw an Error here if the execute is neither Pandas nor SQL.